### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by CindyDorantes**
## Project requirements 🎯  🎯 
- [x]  Create a method called `current_user` in your ApplicationController to make current user data available to all controllers.
    -  It will return the first user from the database.
    -  Don't worry about proper authentication at this time. You will add it in the upcoming projects.
- [x] Create forms to perform the following functions:
    - Creates a Post on behalf of the `current_user` (use the method that you created in your ApplicationController).
    - Create a comment on behalf of the `current_user` (use the method that you created in your ApplicationController).
- [x] Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      # format.html { render :new, locals: { comment: comment, post_id: post_id } }
+      format.html { render :new, locals: { comment: comment } }
+    end
+  end
+
+  def create
+    # new object from params
+    post = Post.find(params[:id])
+    comment = Comment.new(text: params[:comment][:text], post: post, author: current_user)
+    # respond_to block
```
Comment (codepantha) [inline app/controllers/comments_controller.rb:15]: - [ ] An error occurs while trying to create a comment for a post. Kindly view the image attached below to see the error 👇 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @CindyDorantes 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ No linter errors
✔️ Used GitFlow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (CindyDorantes) [reply app/controllers/comments_controller.rb]: Done! Thanks

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**